exchange/exchange_test.go

```diff
@@ -1179,9 +1186,12 @@ func TestCategoryDedupe(t *testing.T) {
 
 		adapterBids[bidderName1] = &seatBid
 
-		bidCategory, adapterBids, err := applyCategoryMapping(nil, requestExt, adapterBids, categoriesFetcher, targData)
+		bidCategory, adapterBids, rejections, err := applyCategoryMapping(nil, requestExt, adapterBids, categoriesFetcher, targData)
 
 		assert.Equal(t, nil, err, "Category mapping error should be empty")
+		assert.Equal(t, 2, len(rejections), "There should be 2 bid rejection messages")
+		assert.Regexpf(t, regexp.MustCompile(`bid rejected \[bid ID: bid\_id(1|3)\] reason: Bid was deduplicated`), rejections[0], "Rejection message did not match expected")
```

Comment:
It probably doesn't hurt anything, but the `_` character shouldn't need to be escaped in the regex.

Reply:
Removed the character escape